### PR TITLE
Zer 382 create sitemap html

### PR DESCRIPTION
**Pull request by sofia2123ewwr**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [x] @loqimean
- [x] @VasylenchukMischa

### Second Level Review

- [x] @loqimean
- [x] @VasylenchukMischa

## Summary of issue

Create sitemap.html

## Summary of change

Created page sitemap, add translates

## Testing approach

ToDo

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=95%
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [x]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

File: app/views/sitemap/index.html.slim [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+          li.mb-2
+            = link_to t('.support'), t('.link_to_support'), class: "text-white"
+      .col-md-6
+        h3.mb-4.text-success = t '.calculators'
+        ul.list-unstyled.mb-4
+          li.mb-2
+            = link_to t('.calculator-diapers'), calculator_url, class: "text-white"
+            ul
+              li.ml-2.my-2
+                = link_to t('.less'), t(".link_less"), class: "text-success"
```
Comment (mykhailovasylenchuk) [inline app/views/sitemap/index.html.slim:21]: new line

Comment (sofia2123ewwr) [reply app/views/sitemap/index.html.slim:21]: add

File: config/locales/uk/uk.yml [hunk 2]
```diff
@@ -67,6 +67,19 @@ uk:
...
+      visit_website: "Наш сайт"
+      link_to_website: "https://zerowastelviv.org.ua/"
+      homepage: "Головна сторінка"
+      support: "Підтримати проект"
+      link_to_support: "https://zerowastelviv.org.ua/pidtrymaty/"
+      about_us: "Про нас"
+      calculator-diapers: "Калькулятор підгузків"
+      calculators: "Калькулятори"
+      less: "Як зменшити використання"
+      link_less: "https://zerowastelviv.org.ua/zero-waste-nappies/"
```
[the remark this answers is not in the document]
Comment (sofia2123ewwr) [reply config/locales/uk/uk.yml:82]: delete

File: spec/requests/sitemap_spec.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe "Sitemaps", type: :request do
+  describe "GET /index" do
+    it "returns http success" do
+      get sitemap_path
+      expect(response).to have_http_status(:success)
```
[the remark this answers is not in the document]
Comment (sofia2123ewwr) [reply spec/requests/sitemap_spec.rb:7]: rewrite

File: spec/requests/sitemap_spec.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe "Sitemaps", type: :request do
+  describe "GET /index" do
+    it "returns http success" do
+      get sitemap_path
```
[the remark this answers is not in the document]
Comment (sofia2123ewwr) [reply spec/requests/sitemap_spec.rb:6]: added line

File: app/views/sitemap/index.html.slim [hunk 5]
```diff
@@ -0,0 +1,14 @@
+div.container.list-unstyled.text-success
```
Comment (mykhailovasylenchuk) [inline app/views/sitemap/index.html.slim:1]: Тут дів не треба

**Review by mykhailovasylenchuk — APPROVED** [review]

File: spec/requests/sitemap_spec.rb [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+RSpec.describe "Sitemaps", type: :request do
+  describe "GET /index" do
+    it "is successful" do
+      get sitemap_path
+
+      expect(sitemap_path).to eq("/en/sitemap")
+      expect(response.body).to include("Sitemap")
+      expect(response).to render_template(:index)
+      expect(response).to be_successful
```
Comment (loqimean) [inline spec/requests/sitemap_spec.rb:11]: це перше має бути

File: spec/requests/sitemap_spec.rb [hunk 7]
```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "Sitemaps", type: :request do
+  describe "GET /index" do
+    it "is successful" do
+      get sitemap_path
+
+      expect(sitemap_path).to eq("/en/sitemap")
```
Comment (loqimean) [inline spec/requests/sitemap_spec.rb:8]: а це ще навіщо

File: spec/requests/sitemap_spec.rb [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+      expect(response.body).to include("Sitemap")
+      expect(response).to render_template(:index)
```
Comment (loqimean) [inline spec/requests/sitemap_spec.rb:10]: це друге

File: spec/requests/sitemap_spec.rb [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+        expect(root_path).to eq("/en")
+        expect(response.body).to include("Sitemap")
+
+        get sitemap_path(locale: :uk)
+
+        expect(sitemap_path).to eq("/uk/sitemap")
+        expect(response.body).to include("Мапа сайту")
+        expect(response).to render_template(:index)
+        expect(response).to be_successful
```
Comment (loqimean) [inline spec/requests/sitemap_spec.rb:26]: те ж, що і зверху

File: app/views/sitemap/index.html.slim (hunk 5 above)
Comment (loqimean) [reply app/views/sitemap/index.html.slim:1]: ага

Comment (sofia2123ewwr) [reply app/views/sitemap/index.html.slim:1]: delete

**Review by loqimean — APPROVED** [review]

**Review by mykhailovasylenchuk — APPROVED** [review]

**Pull request merged**